static/js/message_store.js

```diff
@@ -174,7 +174,7 @@ exports.add_message_metadata = function (message) {
 };
 
 exports.clear = function clear() {
-    this.stored_messages = {};
```

Comment:
Let's remove this function.  I did `git blame` and it appears to have been short-lived in its usefulness.